railties/lib/rails/commands/db/system/change/change_command.rb

```diff
@@ -8,7 +8,7 @@ module Command
     module Db
       module System
         class ChangeCommand < Base # :nodoc:
-          class_option :to, desc: "The database system to switch to."
+          class_option :to, desc: "The database system to switch to. (options: #{Rails::Generators::Database::DATABASES.join('/')})"
```

Comment:
Should we use the `enum` option here? This would show the valid values in the help and would also raise an error if you don't pass a valid one. Making possible for us to remove the check from inside the `ChangeGenerator`.

Reply:
I have no idea about `enum` option. Thanks for suggestion.